DuckDuckGo/TabViewController.swift

```diff
@@ -264,13 +264,15 @@ class TabViewController: UIViewController {
         } else {
             tabModel.link = nil
         }
+        tabModel.sessionStateData = (try? webView.sessionStateData()) ?? nil
     }
         
     @objc func onApplicationWillResignActive() {
         shouldReloadOnError = true
     }
 
-    func attachWebView(configuration: WKWebViewConfiguration, andLoadRequest request: URLRequest?, consumeCookies: Bool) {
+    func attachWebView(configuration: WKWebViewConfiguration, sessionStateData: Data? = nil,
```

Comment:
Hey 👋
Just a quick question, why putting a `nil` default value there? Because you wanted to avoid changes when no `sessionStateData` is injected?